tests/src/Kernel/ValueObject/ImageValueObjectTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel\ValueObject;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+use Drupal\oe_bootstrap_theme\ValueObject\ImageValueObject;
+use Drupal\Tests\token\Kernel\KernelTestBase;
+
+/**
+ * Test image value object with image field type. Extracted from oe_theme.
+ *
+ * @see https://github.com/openeuropa/oe_theme/blob/3.x/tests/src/Kernel/ValueObject/ImageTest.php
+ */
+class ImageValueObjectTest extends KernelTestBase {
+  private const ALT_TITLE = 'This is an alternative title';
+  private const TITLE = 'This is a Title';
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'image',
+    'file',
+    'entity_test',
+    'field',
+  ];
+
+  /**
+   * Entity object.
+   *
+   * @var \Drupal\Core\Entity\EntityInterface
+   */
+  private $entity;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->installEntitySchema('file');
+    $this->installEntitySchema('entity_test');
+
+    $this->installSchema('file', ['file_usage']);
+
+    $this->installConfig(['field', 'system']);
+
+    // Create a field with settings to validate.
+    $field_storage = FieldStorageConfig::create([
+      'entity_type' => 'entity_test',
+      'field_name' => 'field_image',
+      'type' => 'image',
+    ]);
+    $field_storage->save();
+    FieldConfig::create([
+      'field_storage' => $field_storage,
+      'bundle' => 'entity_test',
+    ])->save();
+
+    // Copy file in public files to use it for styling.
+    \Drupal::service('file_system')->copy(__DIR__ . '/../../../fixtures/example_1.jpeg', 'public://example_1.jpg');
+
+    // Create image file.
+    $image = File::create([
+      'uri' => 'public://example_1.jpg',
+    ]);
+    $image->save();
+
+    // Create an image item.
+    $alt = self::ALT_TITLE;
+    $title = self::TITLE;
```

Comment:
Aside from the fact that we are going to remove the constants, why do we have these variables declared, while we could just put them in line 82 and 83? These were useful in oe_theme as the values were random.
Let's drop them and use static values directly.